libs/csm/rest/csm_rest_quota.py

```diff
@@ -40,9 +40,13 @@ def __init__(self):
         self.iam_user = None
         self.csm_user = RestCsmUser()
         self.cryptogen = SystemRandom()
+        self.bucket = "iam-user-bucket-" + str(int(time.time_ns()))
+        self.obj_name_prefix = "created_obj"
+        #self.obj_name = "{0}{1}".format(self.obj_name_prefix, time.perf_counter_ns())
```

Comment:
If this is not required, please delete it

Reply:
done